Solutions/ImpervaCloudWAF/Analytic Rules/ImpervaAbnormalProtocolUsage.yaml

```diff
@@ -0,0 +1,30 @@
+id: 363307f6-09ba-4926-ad52-03aadfd24b5e
+name: Imperva - Abnormal protocol usage
+description: |
+  'Detects abnormal protocol usage.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ImpervaWAFCloudAPI
+    dataTypes:
+      - ImpervaWAFCloud

```

Comment:
Since you are using the parser in queries, you will have to add its schema in json format under [CustomTable folder](https://github.com/Azure/Azure-Sentinel/tree/master/.script/tests/KqlvalidationsTests/CustomTables). Only then the validations would pass. Thank You.